source/en.json

```diff
@@ -33,16 +24,11 @@
     "The :attribute must contain at least one symbol.": "The :attribute must contain at least one symbol.",
     "The :attribute must contain at least one uppercase and one lowercase letter.": "The :attribute must contain at least one uppercase and one lowercase letter.",
     "The given :attribute has appeared in a data leak. Please choose a different :attribute.": "The given :attribute has appeared in a data leak. Please choose a different :attribute.",
-    "This action is unauthorized.": "This action is unauthorized.",
```

Comment:
I would not like translations that worked perfectly in previous versions of this package to be removed like this